Description: change data location to /usr/share/1finger-by-kewlers/
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2019-11-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/rubber1.0/Rubber.h
+++ b/rubber1.0/Rubber.h
@@ -42,7 +42,7 @@
 
 #define PI 3.14159265358979
 
-#define DATAPATH "./data/"
+#define DATAPATH "/usr/share/1finger-by-kewlers/data/"
 
 //typedef unsigned int uint;
 typedef void (*fPtr)();
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 CXX = g++
 CFLAGS = -O2 -ffast-math -fomit-frame-pointer -Irubber1.0
 CXXFLAGS = -O2 -ffast-math -fomit-frame-pointer -Irubber1.0 -std=c++0x
-LIBS = ./libbass.so -lglut -lGLU -lGL -lm
+LIBS = /usr/lib/$(shell dpkg-architecture -qDEB_HOST_MULTIARCH)/libbass.so -lglut -lGLU -lGL -lm
 
 OBJ = test3d.o Kewlmation.o \
       rubber1.0/Rubber.o rubber1.0/Shader.o rubber1.0/Texture.o \
